arkane/gaussian.py

```diff
@@ -373,18 +376,31 @@ def load_scan_energies(self):
                     # rigid scans will only not optimize, so just append every time it finds an energy.
                     if rigid_scan:
                         vlist.append(energy)
+                        angle.append(scan_angle * scan_number)
+                        scan_number += 1
                 # We want to keep the values of energy that come most recently before
                 # the line containing "Optimization completed", since it refers
                 # to the optimized geometry
                 if 'Optimization completed' in line:
                     vlist.append(energy)
+                    angle.append(scan_angle * scan_number)
+                    scan_number += 1
+                if 'Optimization stopped' in line:
+                    # failed to converge, skipping and warning
+                    scan_number += 1
+                    failed_scans += 1
                 line = f.readline()
 
         # give warning in case this assumption is not true
         if rigid_scan:
             print('   Assuming', os.path.basename(self.path), 'is the output from a rigid scan...')
-
+        if failed_scans > 0:
+            logging.warning('{0} failed optimization for scan located at {1}'.format(failed_scans, self.path))
+        if scan_number != self._load_number_scans() + 1:  # + 1 since Gaussian runs an additional scan at end.
+            logging.warning('Read {0} scan energies. Expected {1} from input. Scan job may not have completed.'
```

Comment:
perhaps we should raise an error here instead of warn the user

Reply:
I am hesitant to throw an error since the result still may be accurate.

But I am biased. I have a scan that ran out of time on the last calculation, and still is accurately fit. I wouldn't want my whole model to not appear because of this.